pkg/services/accesscontrol/accesscontrol.go

```diff
@@ -4,11 +4,13 @@ import (
 	"context"
 
 	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/registry"
 )
 
 type AccessControl interface {
+	registry.CanBeDisabled
 	Evaluator
-	Store
+	GetUserPermissions(ctx context.Context, user *models.SignedInUser, roles []string) ([]*Permission, error)
 }
 
 type Evaluator interface {
```

Comment:
Do we really need this abstraction? 🤔 Can't we add `Evaluate(ctx context.Context, user *models.SignedInUser, permission string, scope ...string) (bool, error)` to `AccessControl` . It seems `Evaluator` does not really provide anything beside a second level of indirection 🤔

Reply:
Yes, I thought about that also.